bin/cheribsdtest/cheribsdtest_bounds_varargs.c

```diff
@@ -44,6 +48,7 @@
 
 #include "cheribsdtest.h"
 
+#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Move the file to be under `.if ${MACHINE_ABI:Mpurecap}` rather than `#ifdef`ing out the whole file?